packages/preview-middleware/src/base/flp.ts

```diff
@@ -390,6 +418,27 @@ export class FlpSandbox {
             }
         };
     }
+
+    /**
+     * Gets the UI5 libs dependencies from manifest.json.
+     *
+     * @param manifest application manifest
+     * @returns UI5 libs that should preloaded
+     */
+    private getUI5Libs(manifest: Manifest): string {
+        if (manifest['sap.ui5']?.dependencies?.libs) {
+            const libNames = Object.keys(manifest['sap.ui5'].dependencies.libs);
+            return libNames
+                .filter((key) => {
+                    return UI5_LIBS.some((substring) => {
+                        return key === substring || key.startsWith(substring + '.');
+                    });
+                })
+                .join(',');
+        } else {
+            return '';
```

Comment:
@tobiasqueck Should we add default libs for this case? E.g. `sap.ushell, sap.ui.core, sap.m`?

Reply:
Done with https://github.com/SAP/open-ux-tools/pull/1558/commits/744f212637cc807847f14d3d6152d44fa32a1963